api/api/serializers/audio_serializers.py

```diff
@@ -114,6 +114,41 @@ class Meta:
 AudioHyperlinksSerializer = get_hyperlinks_serializer("audio")
 
 
+class AudioAltFileSerializer(serializers.Serializer):
```

Comment:
I think this can be defined as a subclass of `AudioSerializer` with different `Meta.fields`. I think it's worth exploring that possibility to reduce the code as the fields will not have to be defined manually.

Reply:
I tried fixing this in this PR, but got confused because making this a subclass of `AudioSerializer` would make it a recursive serializer. I'll open a new issue for this.